### DISCUSSION TITLE
QField for iOS

### DISCUSSION THREAD

**Discussion by saguaroland1939** [opengisch/QField#2680] [Q&A]
Good morning,

My Meetup group had an absolute blast using QField/QFieldCloud to do our annual volunteer mapping work. Beautiful user interface. I'll post some more QField feedback from that once I finish verifying the data we collected, but I just noticed that my TestFlight beta version expired on my iPhone. I tried the link to re-download it and it is saying new testers aren't being accepted right now. I also don't see the app yet on the Apple Store. Does anyone know what the status is for QField for iOS and if it is possible to use even a beta version right now so I can finish this project?

Thanks so much,
Veronica

**Comment by mbernasocchi** [comment]
Hi Veronica, unfortunately Apple took very long to start reviewing QField 2.0. 
QField is now available on testflight (qfield.org/get).

enjoy
